scripts/kfctl.sh

```diff
@@ -15,9 +15,14 @@ WHAT=$2
 
 ENV_FILE="env.sh"
 
+KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"}
+KUBEFLOW_DEPLOY=${KUBEFLOW_DEPLOY:-true}
+K8S_NAMESPACE=${K8S_NAMESPACE:-"kubeflow"}
+KUBEFLOW_CLOUD=${KUBEFLOW_CLOUD:-"minikube"}
```

Comment:
Can we get rid of KUBEFLOW_CLOUD? This should be provided by the parameter --platform passed on kfctl init

Reply:
removed.